homeassistant/components/vitrea/strings.json

```
@@ -0,0 +1,67 @@
+{
+  "title": "Vitrea integration",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to Vitrea",
+        "description": "Enter the connection details for your Vitrea device.",
+        "data": {
+          "host": "Host",
+          "port": "Port"
+        },
+        "data_description": {
+          "host": "IP address or hostname of the Vitrea device",
+          "port": "Port number (default: 11502)"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Failed to connect to the Vitrea device. Please check the host and port.",
+      "invalid_host": "Please enter a valid host address.",
+      "invalid_port": "Please enter a valid port number.",
+      "unknown": "Unexpected error occurred."
+    },
+    "abort": {
+      "already_configured": "Device is already configured"
+    }
+  },
+  "entity": {
+    "number": {
+      "timer_minutes": {
+        "name": "Timer"
+      }
+    },
+    "switch": {
+      "switch_with_timer": {
+        "name": "Switch with timer (boiler)"
+      }
+    },
+    "cover": {
+      "blind": {
+        "name": "Blind"
+      }
+    }
+  },
+  "services": {
+    "set_timer": {
+      "name": "Set timer",
+      "description": "Set the timer value for a switch with timer functionality.",
+      "fields": {
+        "entity_id": {
+          "name": "Entity",
+          "description": "Timer switch entity to control."
+        },
+        "minutes": {
+          "name": "Minutes",
+          "description": "Timer duration in minutes (0-120)."
+        }
+      }
+    }
+  },
+  "issues": {
+    "outdated_firmware": {
+      "title": "Device firmware is outdated",
+      "description": "Your Vitrea device firmware version {current_version} is below the minimum required version {min_version}. To fix this issue: 1) Open the Vitrea mobile app, 2) Navigate to device settings, 3) Select 'Update Firmware', 4) Wait for the update to complete, then 5) Restart Home Assistant."
```

This can be formatted using markup:

```suggestion
      "description": "Your Vitrea device firmware version {current_version} is below the minimum required version {min_version}. To fix this issue:\n1. Open the Vitrea mobile app.\n1. Navigate to device settings.\n1. Select 'Update Firmware'.\n1. Wait for the update to complete.\n1. Restart Home Assistant."
```

Numbering will be automatic.